wraps/utils/tests/e2e.spec.ts

```diff
@@ -237,6 +239,83 @@ describe("Ethereum Wrapper", () => {
         if (!response.ok) throw response.error;
         expect(response.value).toEqual("0x48656c6c6f03");
       });
+
+      it("should encode packed [address, uint]", async () => {
+        const response = await client.invoke<string>({
+          uri,
+          method: "solidityPack",
+          args: {
+            types: ["address", "uint"],
+            values: ["0x8ba1f109551bd432803012645ac136ddd64dba72", "45"],
+          },
+        });
+        if (!response.ok) throw response.error;
+        expect(response.value).toEqual("0x8ba1f109551bd432803012645ac136ddd64dba72000000000000000000000000000000000000000000000000000000000000002d");
+      });
+
+      it("should encode packed [address[], uint]", async () => {
+        const types = ["address[]", "uint"];
+        const values = ["0x8ba1f109551bd432803012645ac136ddd64dba72,0x8ba1f109551bd432803012645ac136ddd64dba71", "45"];
+
+        const response = await client.invoke<string>({
+          uri,
+          method: "solidityPack",
+          args: { types, values },
+        });
+        if (!response.ok) throw response.error;
+
+        const expected = ethers.utils.solidityPack(
+          types,
+          [["0x8ba1f109551bd432803012645ac136ddd64dba72","0x8ba1f109551bd432803012645ac136ddd64dba71"],
+            "45"]
+        );
+        expect(response.value).toEqual(expected);
+      });
+
+
+      it("should encode packed ethers", async () => {
+        const types: string[] = [
+          "string", "address", "bool",
+          "uint8", "uint16", "uint32", "uint64", "uint128", "uint256", "uint",
+          "int8", "int16", "int32", "int64", "int128", "int256", "int",
+          "bytes", "bytes8", "bytes16", "bytes24", "bytes32"
+        ];
+        const exampleValues: Record<string, string> = {
+          "string": "hello world",
+          "address": "0x742d35Cc6634C0532925a3b844Bc454e4438f44e",
+          "bool": "true",
+          "uint8": "255",
+          "uint16": "65535",
+          "uint32": "4294967295",
+          "uint64": "18446744073709551615",
+          "uint128": "340282366920938463463374607431768211455",
+          "uint256": "115792089237316195423570985008687907853269984665640564039457584007913129639935",
+          "uint": "115792089237316195423570985008687907853269984665640564039457584007913129639935",
+          "int8": "127",
```

Comment:
Can you also add tests for negative numbers since you described that it can cause issues?

Reply:
Yes, 100%. I just added another test that has a test case with a negative number for each int type